web-src/src/locales/de.json

```diff
@@ -0,0 +1,549 @@
+{
+  "data": {
+    "kind": {
+      "file": "Datei",
+      "url": "URL",
+      "spotify": "Spotify",
+      "pipe": "Stream"
+    }
+  },
+  "dialog": {
+    "cancel": "Cancel",
+    "add": {
+      "rss": {
+        "add": "Hinzufügen",
+        "cancel": "Abbrechen",
+        "help": "Hinzufügen eines Podcasts führt zur Anlage einer RSS-Playlist, so kann Owntone das Podcast-Abo verwalten.",
+        "placeholder": "https://url-to-rss",
+        "processing": "Verarbeite…",
+        "title": "Füge Podcast RSS-Feed-Url hinzu"
+      },
+      "stream": {
+        "add": "Hinzufügen",
+        "cancel": "Cancel",
```

Comment:
Not "Abbrechen"?

Reply:
oh, indeed. Should be "Abbrechen"

Reply:
and when scrolling through, I find some other mistakes as well - gonna correct them instantly.